ouroboros-network-framework/src/Ouroboros/Network/ConnectionManager/Core.hs

```diff
@@ -554,6 +554,8 @@ withConnectionManager
     => ConnectionManagerArguments handlerTrace socket peerAddr handle handleError version versionData m
     -> ConnectionHandler  muxMode handlerTrace socket peerAddr handle handleError (version, versionData) m
     -- ^ Callback which runs in a thread dedicated for a given connection.
+    -> PeerSharing
+    -- ^ Own PeerSharing value
```

Comment:
it's surely own value, the we don't have access to remote peer sharing value when we start connection manager.  I'd use `configuration PeerSharing value` instead, then we more precisely communicate where it's coming from.

Reply:
I left a TODO.

Reply:
we already get it from somewhere else, at the same level so I thought it didn't make much sense to wrap it inside `ConnectionManagerArguments`